zombies.js

```diff
@@ -55,6 +83,58 @@
  * -----------------------------
  */
 
+function Player(name,health,strength,speed){
+
+    this.name=name;
+    this.health=health;
+   this.strength=strength;
+   this.speed=speed;
+   this.isAlive=true;
+   this._pack=[];
+   this._MaxHealth=health;
+   this.equipped=false;
+   this._getPack=function (){
```

Comment:
This function isn't necessary since you set the same function on the prototype below.

The same applies to _getMaxHealth.